src/families/polkadot/js-getTransactionStatus.js

```diff
@@ -290,9 +286,9 @@ const getTransactionStatus = async (a: Account, t: Transaction) => {
 
   if (
     t.mode === "bond" &&
-    a.spendableBalance.minus(totalSpent).lt(WARNING_FEW_DOT_LEFTOVER)
+    a.spendableBalance.minus(totalSpent).lt(FEES_SAFETY_BUFFER)
   ) {
-    warnings.amount = new PolkadotAllFundsWarning();
+    errors.amount = new NotEnoughBalance();
```

Comment:
Why did you change this warning into an error ?
We don't want this behaviour i think.